wkhtmltox.c

```diff
@@ -24,12 +24,15 @@
 
 #include "php.h"
 #include "php_ini.h"
+#include "SAPI.h"
 #include "ext/standard/info.h"
 #include "php_wkhtmltox.h"
 
 #include "src/pdf.h"
 #include "src/image.h"
 
+#define WKHTMLTOX_USABLE (wkhtmltopdf_extended_qt() || !strcmp(sapi_module.name, "cli"))
```

Comment:
I think this condition is wrong ...

If we have extended qt then any SAPI is fine.

If we don't have extended qt then probably cli is the only reasonable SAPI - but this condition would stop it working in that circumstance.